### ISSUE TITLE
Invalid signature exception while executing cbioportal-core jar

### ISSUE THREAD

**Issue by forus** [cBioPortal/cbioportal-core#12]
Steps to reproduce:

1. Build the jar with `mvn clean package -DskipTests`
2. Run `java -cp target/core-1.0.5-SNAPSHOT.jar org.mskcc.cbio.portal.util.VersionUtil`

Expected result:

This command should print the cbio db schema version or fail to connect to the database if `application.properties` is not provided.

Actual result: 

```
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.SecurityException: Invalid signature file digest for Manifest main attributes
	at java.base/sun.security.util.SignatureFileVerifier.processImpl(SignatureFileVerifier.java:340)
	at java.base/sun.security.util.SignatureFileVerifier.process(SignatureFileVerifier.java:282)
	at java.base/java.util.jar.JarVerifier.processEntry(JarVerifier.java:320)
	at java.base/java.util.jar.JarVerifier.update(JarVerifier.java:232)
	at java.base/java.util.jar.JarFile.initializeVerifier(JarFile.java:760)
	at java.base/java.util.jar.JarFile.ensureInitialization(JarFile.java:1070)
	at java.base/java.util.jar.JavaUtilJarAccessImpl.ensureInitialization(JavaUtilJarAccessImpl.java:42)
	at java.base/jdk.internal.loader.URLClassPath$JarLoader$2.getManifest(URLClassPath.java:852)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:848)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at java.base/sun.launcher.LauncherHelper.loadMainClass(LauncherHelper.java:797)
	at java.base/sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:692)
```

**Comment by forus** [comment]
@sheridancbio Hi Robert, this issue started when `cbioportal-version` was switched to `v6.0.0`. Do you have any idea why?

**Comment by forus** [comment]
This issue could be solved by broader improvement - removing `cbioportal` from dependencies of `cbioportal-core`.

**Comment by acoffman** [comment]
I ran into this same issue and, while not optimal, was able to resolve it by running the following:

```
  zip -d core-1.0.5-SNAPSHOT.jar META-INF/BC1024KE.DSA
  zip -d core-1.0.5-SNAPSHOT.jar META-INF/BC1024KE.SF
  zip -d core-1.0.5-SNAPSHOT.jar META-INF/BC2048KE.DSA
  zip -d core-1.0.5-SNAPSHOT.jar META-INF/BC2048KE.SF
```

effectively removing the signature files from the `jar`. However I then ran into an issue where the compiled classes from this repo were too new and not compatible with the java version that the current prod version of cbioportal runs on. Changing the target version in this build to 11 in order to be compatible results in it not being able to resolve all the dependencies and the build fails.
